sync-core/src/main/java/com/cloudant/sync/query/UnindexedMatcher.java

```diff
@@ -0,0 +1,112 @@
+//  Copyright (c) 2014 Cloudant. All rights reserved.
```

Comment:
I'd like to leave it since it is already referenced in the code https://github.com/cloudant/sync-android/pull/77/files#diff-80ec4d1d12847da53428d6c14a88dbb7R139 .  I was adding TODO comments everywhere and I missed that here.  If its OK with you I'd like to add a TODO comment here and leave it in place since it will likely be the very next thing to be worked on.  The alternative is to remove the code referenced above.